lib/livebook/session/data.ex

```diff
@@ -50,7 +51,8 @@ defmodule Livebook.Session.Data do
           runtime: Runtime.t(),
           smart_cell_definitions: list(Runtime.smart_cell_definition()),
           clients_map: %{client_id() => User.id()},
-          users_map: %{User.id() => User.t()}
+          users_map: %{User.id() => User.t()},
+          secrets: list()
```

Comment:
```suggestion
          secrets: list(secret())
```

and we want to move the typespec here, because `Runtime` shouldn't know about secrets :)